homework 08_11/Task3.java

```diff
@@ -0,0 +1,20 @@
+import static java.lang.Math.*;
+import java.util.Date;
+import java.util.Calendar;
+import java.util.Scanner;
+
+class Task3 {
+public static void main(String[] args) {
+Scanner myObj = new Scanner(System.in);
+System.out.println("Please enter the value of degrees Celsius:");
+int celsius = myObj.nextInt();
+
+if (celsius < 0){
+System.out.println("It is " + (32 + celsius * 1.8)+ " degrees Fahrenheit");
+}else if(celsius > 0){
+System.out.println("It is " + (32 + celsius * 1.8) + " degrees Fahrenheit");
+}else if(celsius == 0){
```

Comment:
redundant logic, you can simplify it to: 
System.out.println("It is " + (32 + celsius * 1.8)+ " degrees Fahrenheit");